.github/workflows/run_forklift_build_on_kind.yml

```diff
@@ -33,10 +33,14 @@ jobs:
           path: ~/.cache/bazel
           key: ${{ runner.os }}-bazel-cache
 
+      - run: sudo apt-get install -y nbdkit-plugin-dev
```

Comment:
Please comment that this is a dependency for `stub-images/vddk-test-vmdk/vddk-test-plugin.c`. 
And add this also to the action.yml

Reply:
i'll remove this line from this GH-Action , since we are not building it per PR.